web-ui/src/main/resources/catalog/locales/en-core.json

```diff
@@ -367,5 +367,7 @@
     "expandAllFacetLabel": "Expand",
     "resultXonY": "on",
     "resolveCaptcha": "Resolve the captcha",
-    "resolutions": "Resolution"
+    "resolutions": "Resolution",
+    "loginTitle": "Log in",
+    "loginText": "Log in with your username and password to add and edit metadata."
```

Comment:
We use "sign in" and "sign out" in other places. Would be good to use it here also instead of "log in".

Reply:
Ok, I will change it.